notifications_utils/formatters.py

```diff
@@ -194,6 +194,13 @@ def escape_html(value):
     return value
 
 
+def unescape_html(value):
```

Comment:
I don't think we need this - it isn't used anywhere. Can we drop it now? :)

Reply:
Yes. I will take it out